<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookAppointmentWindow</class>
 <widget class="QDialog" name="BookAppointmentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="slotsListView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>331</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSubmit">
   <property name="geometry">
    <rect>
     <x>259</x>
     <y>277</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script MT Bold</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelERROR">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script MT Bold</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;*Please select a slot.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Available_slots">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Available slots and Doctors</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_speciality">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Script MT Bold</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Speciality</string>
   </property>
   <widget class="QRadioButton" name="radioButton_Nutrition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Nutrition</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_OG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Obstetrics and Gynecology</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Ophthalmology">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>191</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Ophthalmology</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_internalmedicine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>241</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Internal Medicine</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Dermatology">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Dermatology</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
